packages/odyssey-react-mui/src/labs/FileUpload.tsx

```diff
@@ -174,7 +183,7 @@ const FileUpload = ({
   );
 
   const triggerFileInputClick = useCallback(() => {
-    inputRef.current?.focus();
+    inputRef.current?.click();
```

Comment:
Is this different from `.focus`?

Reply:
Not sure what you mean exactly. We need to `click` here to trigger the file picker dialog, if that is what you are looking for. Just `.focus` wasn't working as we'd think